KarhooSDK/Api/Response/PaymentProvider.swift

```diff
@@ -24,7 +24,7 @@ public struct PaymentProvider : KarhooCodableModel {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         
         self.provider = (try? container.decode(Provider.self, forKey: .provider)) ?? Provider()
-        self.version = (try? container.decodeIfPresent(String.self, forKey: .version)) ?? "v51"
+        self.version = provider.version
```

Comment:
This field is not in the API response. Why are we extracting it instead of getting it directly from the provider object when necessary?